src/puppetlabs/dujour/version_check.clj

```diff
@@ -81,7 +81,8 @@
         version-data {:version current-version}
         request-body (-> request-values
                          (dissoc :product-name)
-                         (set/rename-keys {:agent-os :agent_os})
+                         (set/rename-keys {:agent-os :agent_os
+                                           :puppet-agent-versions :puppet_agent_versions})
```

Comment:
Actually, what happens if puppet-agent-versions isn't passed in to this version check? It seems like we get some weird behavior:

```clojure
user=> (def testmap {:agent_os "ohey"})
#'user/testmap
user=> (clojure.set/rename-keys {:agent_os :agent-os :puppet-agent-versions :puppet_agent_versions} testmap)
{:puppet-agent-versions :puppet_agent_versions, "ohey" :agent-os}
```